src/components/FeatureViews/AdminPortal/BulkUploadModal.js

```diff
@@ -106,10 +108,10 @@ const GET_TEMPLATE = {
 };
 
 const BulkUploadModal = ({ closeModal }) => {
-    const [template, setTemplate] = useState('');
+    const [selectedItem, setSelectedItem] = useState('');
```

Comment:
I feel like selectedTemplate would be a better variable name for this. lmk what you think